src/client/content_mapblock.cpp

```diff
@@ -1433,8 +1480,10 @@ void MapblockMeshGenerator::drawNodeboxNode()
 		}
 	}
 
-	for (auto &box : boxes)
-		drawAutoLightedCuboid(box, nullptr, tiles, 6);
+	for (auto &box : boxes) {
+		u8 mask = getNodeBoxMask(box, solid_neighbors, sametype_neighbors);
```

Comment:
one more: this must only be allowed if the node has `alpha == ALPHAMODE_OPAQUE`
otherwise you get different results:
![grafik](https://user-images.githubusercontent.com/1042418/166559557-b54f1363-bc65-4500-ab9f-841ad98f4224.png)
of course this isn't undesirable for glass but it would have to be *optional*
(strictly speaking the documentation does not define what happens to internal faces of nodeboxes so could be worth a discussion)

Reply:
I've pushed a small change to restore the previous behavior, and we can discuss whether and how we want to change it, now that we can control what's rendered.